internal/throttler/sampler/throttled_sampler.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sampler
+
+import (
+	"github.com/uber/jaeger-client-go"
+	"github.com/uber/jaeger-client-go/internal/throttler"
+)
+
+// N.B. this class is private as it's a proof of concept of how we would throttle normal
```

Comment:
As mentioned in the comment, this is just a proof of concept of how we'd throttle normal sampler traffic. (I'll remove before landing).